File: sub-web-model/src/main/java/org/github/base/service/impl/MessageServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.MessageEntity;
import org.github.base.mapper.IMessageMapper;
import org.github.base.service.IMessageService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 *  服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-08
 */
@Service
public class MessageServiceImpl extends ServiceImpl<IMessageMapper, MessageEntity> implements IMessageService {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/service/impl/MessageServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.service.impl;

import org.github.mysql.blog.base.entity.MessageEntity;
import org.github.mysql.blog.base.mapper.IMessageMapper;
import org.github.mysql.blog.base.service.IMessageService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 *  服务实现类
 *
 * @author JYD_XL
 */
@Service
open class MessageServiceImpl : ServiceImpl<IMessageMapper, MessageEntity>(), IMessageService
